lv_freetype.c

```diff
@@ -63,41 +262,41 @@ static bool get_glyph_dsc_cache_cb(const lv_font_t * font, lv_font_glyph_dsc_t *
         return true;
     }
 
-	FT_UInt glyph_index;
-	FT_UInt charmap_index;
-	FT_Face face;
-	lv_font_fmt_freetype_dsc_t * dsc = (lv_font_fmt_freetype_dsc_t *)(font->user_data);
-    face = dsc->face;
-	FTC_ImageTypeRec desc_sbit_type;
-
-	desc_sbit_type.face_id = (FTC_FaceID)face;
-	desc_sbit_type.flags = FT_LOAD_RENDER | FT_LOAD_TARGET_NORMAL;
-	desc_sbit_type.height = dsc->font_size;
-	desc_sbit_type.width = 0;
+    lv_font_fmt_ft_dsc_t * dsc = (lv_font_fmt_ft_dsc_t *)(font->user_data);
+    FT_Face face = dsc->face;
 
-	/*FTC_Manager_LookupFace(cache_manager, face, &get_face);*/
-	charmap_index = FT_Get_Charmap_Index(face->charmap);
-	glyph_index = FTC_CMapCache_Lookup(cmap_cache, face, charmap_index, unicode_letter);
-	FTC_SBitCache_Lookup(sbit_cache, &desc_sbit_type, glyph_index, &sbit, NULL);
+    static FTC_ImageTypeRec desc_sbit_type;
+    desc_sbit_type.face_id = (FTC_FaceID)face;
+    desc_sbit_type.flags = FT_LOAD_RENDER | FT_LOAD_TARGET_NORMAL;
+    desc_sbit_type.height = dsc->weight;
+    desc_sbit_type.width = dsc->weight;
+    FT_UInt charmap_index = FT_Get_Charmap_Index(face->charmap);
+    FT_UInt glyph_index = FTC_CMapCache_Lookup(cmap_cache, (FTC_FaceID)face, charmap_index, unicode_letter);
+    FT_Error error = FTC_SBitCache_Lookup(sbit_cache, &desc_sbit_type, glyph_index, &sbit, NULL);
 
     dsc_out->adv_w = sbit->xadvance;
-    dsc_out->box_h = sbit->height;         /*Height of the bitmap in [px]*/
```

Comment:
The comments will probably be useful for future readers... any reason why they were removed?